### DISCUSSION TITLE
How can I apply room temperature offset

### DISCUSSION THREAD

**Discussion by bryce402** [omerfaruk-aran/esphome_samsung_hvac_bus#177] [General]
I am using m5stack for controlling samsung AC
When I try to add room temperature offset there is an error like below.
when I see example.yaml there is an option room_temperature_offset
is there anyone can help me to solve this error?

     [room_temperature_offset] is an invalid option for [devices]. Did you mean [room_temperature], [target_temperature]?
     room_temperature_offset: -2

**Comment by lanwin** [comment]
Hi, please check if you have `github://lanwin/esphome_samsung_ac@main` instead of `github://lanwin/esphome_samsung_ac@stable`
